platform/uwp/app.h

```diff
@@ -28,7 +28,8 @@
 /* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                */
 /*************************************************************************/
 
-#pragma once
+#ifndef APP_H
+#define APP_H
```

Comment:
This is pretty general, should we rename this file to `app_uwp.h`?

Reply:
Sounds good.